app/src/main/java/com/dozingcatsoftware/vectorpinball/elements/SensorElement.java

```diff
@@ -52,10 +52,7 @@
     boolean ballInRange(Ball ball) {
         Vector2 bpos = ball.getPosition();
         // Test against rect.
-        if (bpos.x < xmin || bpos.x > xmax || bpos.y < ymin || bpos.y > ymax) {
-            return false;
-        }
-        return true;
+        return !(bpos.x < xmin) && !(bpos.x > xmax) && !(bpos.y < ymin) && !(bpos.y > ymax);
```

Comment:
I prefer the original. Combinations of negated conditions are harder to interpret.